### PR TITLE
#w5ev - Send ui events to iTasks

### PR DESCRIPTION
**Pull request by tbeuzenberg**

### CODE REVIEW

File: itasks_server/.gitignore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-*
```
Comment (nickhidding) [inline itasks_server/.gitignore]: This file is needed to keep this directory in the Git repository

File: itasks_server/websocket-sharp.xml [hunk 2]
```diff
@@ -0,0 +1,7850 @@
+<?xml version="1.0"?>
```
Comment (nickhidding) [inline itasks_server/websocket-sharp.xml]: This file and the .exe do not belong inside this repository

File: qt_event_handler.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    """ QtEventHandler handles the events from QWidgets """
+
+    def __init__(self, itasks_service):
+        """
+        :param itasks_service: iTasks service
+        """
+        self.itasks_service = itasks_service
+
+    def button_clicked_event(self, itasks_component: ItasksComponent):
+        """ button clicked event handler """
```
Comment (nickhidding) [inline qt_event_handler.py]: python docstring params are missing (for consistency with other classes)

File: itasks_server/.gitignore (hunk 1 above)
Comment (tbeuzenberg) [reply itasks_server/.gitignore]: readded

File: itasks_server/websocket-sharp.xml (hunk 2 above)
Comment (tbeuzenberg) [reply itasks_server/websocket-sharp.xml]: removed

File: qt_event_handler.py (hunk 3 above)
Comment (tbeuzenberg) [reply qt_event_handler.py]: added params to docstrings

**Pull request merged**